content/200-concepts/100-components/08-prisma-engines/index.mdx

```diff
@@ -47,9 +47,8 @@ You can define environment variables globally on your machine or in the `.env` f
 Add the environment variable to the [`.env` file](../../more/environment-variables#using-env-files).
 
 <TabbedContent
-  tabs={[<FileWithIcon defaultKey="unix" text="Unix, macOS" icon="code"/>, <FileWithIcon defaultKey="windows" text="Windows" icon="display"/>]}
-  defaultTabType="OS"
->
+tabs={[<FileWithIcon defaultKey="unix" text="Unix, macOS" icon="code"/>, <FileWithIcon defaultKey="windows" text="Windows" icon="display"/>]}
```

Comment:
More readable before probably - but not a big deal.

Reply:
@janpio looks like there is a bug with MDX in Prettier, you can see in [repl](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBadAdWBLGAbOJAAgHIAFAJ2wGcBbAQyIQHNso5qSta4Z6AVXAWLkqdRizYcuUHnwAiHMFQAOMbNBEAZOPQpQi9AEYQArjCKUaDIm3j76eaoagATIgAsIAd1sXvEBQA1s4YIKZucBREMB5wnhAQrmEAdJZiNjCJToYU8Ub01HDu0C5MUKzsqHj0AJ5RRN4e2GAeRAxQ9MwcMXFEkLS0ES306qXeuG2x8a6jxoVwKTLoqFhYADz8ECoAQngQYEEAfGtQ-M3OKl3xrABuPfQGEPcUt9hwvhAAZr3xVuK2WBRTo5R7uLy+XCNQIhIhhCKuBrTBJJVKnAByEHgvVGRAAVHjYjQiFdukRXBAelAsf1oHw2C5aiSKPQwOowI5AV9AgwxgZYrjiXkCLdHhZudEAPqmahsZiS9LWegEtIASQsLOm0Xo2FozlxKgoz2wrjlhiIkp4sE5tCScDwColjQ8uOR-0y2SCUMlrJgpkceFqCuJYISvmRpq+XyiCAsWQgOQFFj08XoouwNSMBBiEHJcHu+xUUX1eSI1BgFFMbNMeVcKVOqp+tTMJFL7HDufd9AANEQjPFqDX4ll+nFDkQzHG+gBtACKphaIT4FBgAF0ABQpFIAei3u53e+6MHUFVQ5b08Fc24AjgvDueVwBKFzuaeq2BG1xVvkbvd755RG8Hzbk0oyoDQqCGkqz6kj0XzYBQ5b1lAGzblsuz7IcJwoVAADEuGKgCkjsNQpwAIJTvEkClt8TCsm0tpfgQvZasOtQqCMeCBrY+pENOXblJUcAbh4x4qNQSDbturCxKYRgpAM25QeISkZPQqDERwz5JrYtAqAQVowM4yLUQOvATj8XwRGyGggrgtRpAAohUUglvEur6XAhnFICfEAEoymu66iTA4mSSB3jeCklbltUjzMCkgTMNuz6hnAAAeKgQEU5r7N41T5vaRBkWQqo4hYxIyj5RhMsizTMHEFAFQWgL2F8rIcMhWBkYRNiaTxvz4nipp5GyrVRO1YDDrmEZzAURQEr2jxMvVjXNUVdgTR1+p4N4dTOAMQxQCM2ITLEg2zHw83xJKsRGqYDUKsimlxfUFBdVAZH6gYGX0HpzG9YwADCeDvLAtJQOwbLGTNM63lETKab+B47gMWXsLA1CqUqGkuSRN6mAjuNCc+DKBIi0Qjnk9DuKG3hUNil2MAyjBM9dSCnAAhPxamCVIIliRJUnYCkurMDWCkQLQ26qgAHBSABMACsZEpCoFSPqc+EEQAqrKFT9IFUt8+wfZsHo7ykThOxMoi7WmHgMBLVxZudFQDylvQ5hS6MnHcRS3hQPsNM+XYsPxAABtSiKWkkDscNuAACykMNumnUBHRDcngFPOggRDNqYgLni7gREKYKiXZHKf0BHLF9AJQNaGVVyHNcaTruc8TTvDFCI3jwmbij25o9AsZYzXxOuQTROaaTO3ULmkAcT5I7It07Caj5jeg7GecGIXRAb1EoyLM+ACaZjtNgDX+GKOblzljDTmARu0K7ei1ALoVC9JkxyZLaWNdsbiCniRZ88FszYC+BzHCqBire15D5AOQcIA0wsh-d2zhiTUgsFlagsosyLCwPAy+RcXT3H6NTS8BczDRB8D9AemCmTridPAcscpexlydI8IgAB5AAyuVAauCLLwTANgAMTJBwqCyiuYoj4Pp62HH0bOeUzQIDeEaWQe9RRUGMAQGGZYiwSK+EyV+5ZjaYV9tAZwTpC7RCMObLBsCSF8QjmQPyqoBEAFkyKSlnDrRyflz6SkcuiAA4qqdEjlJQ7GiWREJEdkb7lSXuPI0Y8hQCmtuegHFUAZJjNkuAacoBaOgIZVAejJFEOoAUuAmSEA5NwjXSUvcgyaUlE4t2tRnzrGoFcAwYAagEIALxhDydgMIRx1jbgGY8I4RB1zzgRibOAmsoDwOnB4rxvj-E+NVBEvyZF+Cqj4eiMJkTomxPieiRJ59klDzScPQpWScmTPqY04ppTyk6NgFUi2BiOCfKKc01ptAb6alspKTp3TP7PiIP0wZ-QRnUHGSASZ0zZnzKgIs9cPjIU2MYSTNx2zPHeL8ZKaJ-A-KCLII5IGpzzmXKiTEuJCSkkpL-C8hpoKSkfNeU0kpmiEIVNjAC-RtSQVvJKS0tSko7BGgGXAGy0AYUDy6c43piKcUosKGiiZHEsVzMGXi98FZsomL5GsjZWydkUv8QAMR8fwdldzOVPO5akwV3yBW8plT80VfyYASpqYY6VQrtxyqVJKL4tAYCap6X03Vwz9XosxSAGZJqFlLKdFHFQ6VmRKizjyUYEcNlYG1gRARvATzMEGiK7RhkiDVKBacMh-ReF2ykOUX5zbW21KPvsAoXEmSlAce0eiPay4MmRBHFImjM6QOIThKtBF6DPi7kQedi6s4ZjgORVw7hnplKDf2wFRCH7Im2QuspS791cp3LaPIgam3ioHYY3CMo5S41uKgZd1BFGnE2MYfsrggZ0ljFgIgMRjBouANOdYjr90AHVJiqkgAYO2XtHYAGk4C1HRcMdKYQYgZRgOinWx10q9gYGAQRpGWjQHRZAREYRtxHF7Eh1D6HMN5ntnhgj6KJhuB8KREAZH0oUbCGh0T3hxO2Ew+i00Ayai1HY0cVcABfaD-GcMwH4MYfg7E4DooYyALA2ENhXSs1ACO9msDkr2QEoJISWXXLdfc0ZFisjS1oLUVA7SwFwFQMRrA9mI7Ae3DZ6zxhbMRcc7sylgTgmhPCaym5HLz7eY5hgHzUsMAYCuLEQr-nAuEz7sF1AcK+4LvSgeuzDmcKzJi1AWZhmjBgYg0CWAtnFkEkxPAAkxB1QDXwYQ7MI5pxVTKDuu9e6oEGEYNY61k5ZSIkGhHGuS6EwU0fduZ9wrT1vv+R+hOHQug-s0f+-ddSwRnlrT+s7gGPprr7M+CJw7OSNrFeDM7pwAr8lUQmdRBtDqhhHEUSivaz26IvdmZgX3R1LNnRcJg6U-qeV7Pa5zKW3PpY87c+55bXFtY611yDsBdNXXg4h5DBA0OxAw6UbDDsYD4cI-CajpH4BSco9R2jrJzOKeY2EVjB6QAca4-TuAjOPDM6ww0-THPhNsADgp3n0mQCyfV4xpTYQVP6TqBp7TunWeOw68ZosZmBFhEs6cGDiLWvhfs-YCFIIsAZTkRYJzyXXNpauWyonITRkjzfgdgLQWXphca5FnCjuWtxYdzB9YzvY9u-NngLAUPLBJf8XjgPGXPMh7ALl-LtBSuR4qwFl6In1d1Yawl+PKfotJ+a2hUDxRuvwF61rbWRB+AcAsBOn7waW3w4a1gQHg01E+DNODtwD9JwqHMINYrHgjEBkwZbUnEWM8e8hgWotAIqku7j1gLdy-V-UAhMZF00P2lrJJKMNoAw4JGnfnO33+f-fuaD1lzOUfc9SVYQYDcnLvSnGAanODUZBDbjBnXjFnJXNnFXLnbAEjCTTXfndAwXejW3CTJjKAFjO0DTaXHjJnPjc3dnITMIOvMTHncjdFHXeggg-XEAQ3NTE3HTAwPTNnS3EzG3O3XFZPJ3NvM-G-TwG+DwUGW+UZBWAAbmpAoAYCzygCBhrCyTwRqBgAlHfkdyIGIFmAoBEywBWT7iIGciEn0OsOICjyYVcAADYowAB2AARlcIAE4HCAAWDwjwhWBWIwWWbwuAbw1wgAZiMEiIcK8IcIAAY4AFYvhfDEij0vhwillcQw9LE-Mq9Z4B5QtqMNkCVmAoVShLCe19DiAIVSiiUqthlsByQnCvg3DPCfC-CAigiQiwjIjojYiEikiUiFY0iMj1wsj-NDQIAAArFVGAbcaOOAWOJiBOZONSUpISLGGosozZaPIorAc1JVK1WyCwphYgRVS1WY2yVAUyJolw9wrw3w-wwI4I0IiIqIowGIhw+IxI5Ijw1I1wdIzIiwCPSYmYtkeYu0JY+OLGVYpUdY6ec45VVVHYgo4jDZR1UtCwHYLVawx3YgSeONCwRwu4tox4zol4no94z474wYv44YgE0Y8Y9iI0MEuYhYqEwxJOYBdOEBBgf9eNQo9AjZJvXTRFVvIw) that it fails to parse this file, so I had to concat some spaces manually to make it work